.github/workflows/ci.yaml

```diff
@@ -38,3 +38,6 @@ jobs:
     - name: Run tests
       run:
         npm run test
+    - name: Lint code
+      run:
+        npm run lint-errors
```

Comment:
Isn't this run in both pipelines now?

Reply:
For now yes, unless you'd prefer to only have it one?